src/transport_picoquic.cpp

```diff
@@ -271,7 +269,7 @@ pq_loop_cb(picoquic_quic_t* quic, picoquic_packet_loop_cb_enum cb_mode, void* ca
 
             case picoquic_packet_loop_port_update:
                 log_msg << "packet_loop_port_update";
-                transport->logger.log(LogLevel::debug, log_msg.str());
+                LOGGER_DEBUG(transport->logger, log_msg.str());
```

Comment:
the macros are nice but maybe we use them consistently... otherwise maybe not use them.

Reply:
Most log messages we will likely want to keep in release code and enable with a run-time config.  However, debug logs are likely ones we want to compile-out.  That was the reason I used LOGGER_DEBUG. In general, I'm not a fan of the macro form, but I do appreciate the ability to compile out those lines.